mllib/src/test/java/org/apache/spark/mllib/feature/JavaWord2VecSuite.java

```diff
@@ -50,9 +50,9 @@ public void tearDown() {
   @SuppressWarnings("unchecked")
   public void word2Vec() {
     // The tests are to check Java compatibility.
-    String sentence = Strings.repeat("a b ", 100) + Strings.repeat("a c ", 10);
-    List<String> words = Lists.newArrayList(sentence.split(" "));
-    List<List<String>> localDoc = Lists.newArrayList(words, words);
+    String sentence = StringUtils.repeat("a b ", 100) + StringUtils.repeat("a c ", 10);
```

Comment:
There are a few issues here. This is using Commons Lang 2.x and would be better to use 3.x (`commons.lang3.*`). However, Spark doesn't generally depend on either of them; a few modules do but not MLlib. This happens to work because of transitive dependencies. Since we definitely do depend on Guava, and that won't go anywhere soon, and this is an internal use anyway, I think I might leave out this change.